cmd/influxdb-writer/main.go

```diff
@@ -61,13 +62,22 @@ func main() {
 	if err := env.Parse(&influxDBConfig, env.Options{Prefix: envPrefixInfluxdb}); err != nil {
 		logger.Fatal(fmt.Sprintf("failed to load InfluxDB client configuration from environment variable : %s", err))
 	}
-	client, err := influxDBClient.Connect(influxDBConfig)
+	influxDBConfig.DBUrl = fmt.Sprintf("%s://%s:%s", influxDBConfig.Protocol, influxDBConfig.Host, influxDBConfig.Port)
+	if influxDBConfig.Timeout == 0 {
+		influxDBConfig.Timeout = 1000 * time.Millisecond
```

Comment:
You can set the default value in `/internal/clients/influxdb/influxdb.go`like:
```go
type Config struct {
	Protocol           string        `env:"PROTOCOL"              envDefault:"http"`
	Host               string        `env:"HOST"                  envDefault:"localhost"`
	Port               string        `env:"PORT"                  envDefault:"8086"`
	Username           string        `env:"ADMIN_USER"            envDefault:"mainflux"`
	Password           string        `env:"ADMIN_PASSWORD"        envDefault:"mainflux"`
	DbName             string        `env:"DB"                    envDefault:"mainflux"`
	Bucket             string        `env:"BUCKET"                envDefault:"mainflux-bucket"`
	Org                string        `env:"ORG"                   envDefault:"mainflux"`
	Token              string        `env:"TOKEN"                 envDefault:"mainflux-token"`
	DBUrl              string        `env:"DBURL"                   envDefault:""`
	UserAgent          string        `env:"USER_AGENT"            envDefault:"InfluxDBClient"`
	Timeout            time.Duration `env:"TIMEOUT"  envDefault:"1s"` // Influxdb client configuration by default there is no timeout duration , this field will not have fallback default timeout duration Reference: https://pkg.go.dev/github.com/influxdata/influxdb@v1.10.0/client/v2#HTTPConfig
	InsecureSkipVerify bool          `env:"INSECURE_SKIP_VERIFY"  envDefault:"false"`
}
```

Reply:
Also, please align these tags. :)

Reply:
Thanks @dborovcanin , I wasn't able to find this config struct for some reason, hence the use of extra line.